src/index.html

```diff
@@ -5,8 +5,305 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo_img">
+        <img src="https://github.com/mate-academy/layout_moyo-header/blob/master/src/images/logo.png?raw=true" alt="Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#Apple"
+              class="nav__link is-active"
+              data-qa="hover"
+             >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item" >
```

Comment:
Add a blank line between the links for better readability
```suggestion
          </li>
          
          <li class="nav__item" >
```